docs/accessing_resource_localhost.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Accessing your resource on localhost
+layout: docs
+---
+
+## Access your resource with Kubernetes port forwarding
+
+As you're configuring [resources in Tilt](/tiltfile_concepts.html#resources), you'll often want to access an app within a Kubernetes resource, for example, a web app or a database server. Kubernetes provides [port forwarding](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) to allow you to access via your local machine with `localhost`. Tilt exposes this feature through a configuration in [`k8s_resource`](/api.html#api.k8s_resource). For example, if you have in your Tiltfile: 
+
+```
+k8s_resource(workload='blog-site', port_forwards=4002)
+```
+
+`localhost:4002` will be connected to container port 4002. Tilt also displays a link at the top of the screen too.
+
+<figure>
+   <img src="/assets/img/port-forward.png" title="Port forward">
+</figure>
+
+With:
+
+```
+k8s_resource(workload='blog-site', port_forwards='9000:4002')
+```
+
+`localhost:9000` will be connected to container port 4002.
+
+## Multiple port forwards
```

Comment:
FFTI: I might call this section `### Naming Port Forwards` instead, because naming PF's can be useful even if you don't have more than one (and then after you introduce the syntax, have a line that says "this can be especially useful if you have multiple port forwards and want to disambiguate between them" or something).

Reply:
Good idea. Updated the heading. (I already talk about the disambiguating use case in the text.)